kt/godot-library/src/main/kotlin/godot/core/bridge/StringName.kt

```diff
@@ -64,4 +64,6 @@ class StringName : NativeCoreType {
     }
 }
 
-fun String.asStringName() = StringName(this)
+fun String.asStringName(): StringName {
```

Comment:
As mentioned on discord, we'd need this for nodepath as well

Reply:
I've added NodePath and moved the extension function for StringName into StringExt.